pkg/asset/machines/ibmcloud/machines.go

```diff
@@ -0,0 +1,142 @@
+package ibmcloud
+
+import (
+	"fmt"
+
+	ibmcloudprovider "github.com/openshift/cluster-api-provider-ibmcloud/pkg/apis/ibmcloudprovider/v1beta1"
+	"github.com/openshift/installer/pkg/types"
+	"github.com/openshift/installer/pkg/types/ibmcloud"
+	machineapi "github.com/openshift/machine-api-operator/pkg/apis/machine/v1beta1"
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
nitpick: we usually separate the openshift imports from the others... ie:
```
"fmt"

"github.com/pkg/errors"

"github.com/openshift/installer/pkg/types"
```

Reply:
Done